go/stellar/stellarsvc/frontend.go

```diff
@@ -192,6 +192,33 @@ func (s *Server) GetDisplayCurrenciesLocal(ctx context.Context, sessionID int) (
 	return currencies, nil
 }
 
+func (s *Server) GetUserSettingsLocal(ctx context.Context, sessionID int) (userSettings stellar1.UserSettings, err error) {
+	ctx = s.logTag(ctx)
+	defer s.G().CTraceTimed(ctx, "GetUserSettingsLocal", func() error { return err })()
+	err = s.assertLoggedIn(ctx)
+	if err != nil {
+		return stellar1.UserSettings{}, err
```

Comment:
same here, can just `return userSettings, err`, so you don't have to repeat the return type